pages/developer-docs/installation/installing_sunbirdon_laptop.md

```diff
@@ -57,6 +57,8 @@ After checking out the code, execute the following command:
 
     git clone https://github.com/project-sunbird/sunbird-portal.git
 
+Note: The sunbird portal stable versions are available in tags for each release and master branch contains the latest stable release.
```

Comment:
@harishgilimi please add a link or instructions for getting a stable release from a tag. We cannot assume that the reader will know how to get the release from a tag.

Reply:
@kochhar  sure i will add it